phpstan.neon

```diff
@@ -20,6 +20,8 @@ parameters:
 		- *.inc.php
 		- app/bundles/CoreBundle/Controller/AbstractFormController.php
 		- app/bundles/CoreBundle/Controller/AbstractStandardFormController.php
+		- app/bundles/CoreBundle/Entity/TranslationEntityTrait.php
```

Comment:
I had stop checking these files because there is no solution for it at the moment

Reply:
This is not relevant any more, thank for @daniellord32.